CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Document how to create alerts based on logs.
+
+### Changed
+
+- Extra logs and metrics tenant into helm values.
```

Comment:
```suggestion
- Make logs and metrics tenant configurable via helm values.
```

Reply:
Well I wanted to write extract but make is better